posit_team_applications_faq.md

```diff
@@ -22,6 +22,33 @@ If you are working remotely, yes: ensure that when you login to Windows that you
 
 ### Installing Packages
 
+#### How do I install the `{hablar}` package?
+
+The `{hablar}` cannot be installed as a pre-compiled binary; attempting this gives an error.  Therefore, you need to force R to install the source version by specifying the URL for the source version of packages on Package Manager.  However, `{hablar}`'s dependencies can be installed as binaries first.
+
+```r
+# Get list of dependencies and imports for {hablar}
+available_pkgs <- available.packages()
+
+deps <- tools::package_dependencies(
+  packages = available_pkgs["hablar", "Package"],
+  recursive = TRUE)[[1]]
+
+# Install these dependencies as binaries
```

Comment:
When I ran this line I got into the you need to restart loop of death as I'd pressed "yes". Would it be worth mentioning in the notes not to press "yes" for a restart at this point, or is that common knowledge?